src/library-authoring/components/ComponentCard.tsx

```diff
@@ -21,39 +23,47 @@ type ComponentCardProps = {
   blockTypeDisplayName: string,
 };
 
-const ComponentCardMenu = () => (
-  <Dropdown>
-    <Dropdown.Toggle
-      as={IconButton}
-      src={MoreVert}
-      iconAs={Icon}
-      variant="primary"
-    />
-    <Dropdown.Menu>
-      <Dropdown.Item disabled>
-        <FormattedMessage
-          {...messages.menuEdit}
-        />
-      </Dropdown.Item>
-      <Dropdown.Item disabled>
-        <FormattedMessage
-          {...messages.menuCopyToClipboard}
-        />
-      </Dropdown.Item>
-      <Dropdown.Item disabled>
-        <FormattedMessage
-          {...messages.menuAddToCollection}
-        />
-      </Dropdown.Item>
-    </Dropdown.Menu>
-  </Dropdown>
-);
+const ComponentCardMenu = ({ usageKey }: { usageKey: string }) => {
+  const intl = useIntl();
+  const { showToast } = useContext(ToastContext);
+  const updateClipboardClick = () => {
+    updateClipboard(usageKey)
+      .then(() => showToast(intl.formatMessage(messages.copyToClipboardSuccess)))
+      .catch(() => showToast(intl.formatMessage(messages.copyToClipboardError)));
+  };
+
+  return (
+    <Dropdown id="component-card-dropdown">
+      <Dropdown.Toggle
+        id="component-card-menu-toggle"
+        as={IconButton}
+        src={MoreVert}
+        iconAs={Icon}
+        variant="primary"
+        alt="component-card-menu-toggle" // FixMe: Add alt text
```

Comment:
nit: Adding the alt intl text

Reply:
Fixed here: ace4c25caa6cc5eb652919671c221e21726a59e8